### PR TITLE
Settings.py: Remove unused variables 

### PR DESCRIPTION
**Pull request by sjha2000**
Closes https://github.com/coala/coala-quickstart/issues/264

### CODE REVIEW

**Comment by palash25** [comment]
@sjha2000 stop making more commits. First learn how to squash them I have provided you with a link to an article then amend your commit message according to our guidelines and then only push. Other people also use the CI and if you keep pushing every time you make a small (& wrong) change someone else's builds might get queued.

So only push once you have amended your commits according to the reviews

**Comment by palash25** [comment]
Look over here you have made 7 commits in your branch but we only need one https://github.com/coala/coala-quickstart/pull/269/commits

**Comment by sjha2000** [comment]
>>Look over here you have made 7 commits in your branch but we only need one
currently i have no idea what am i doing i was trying to squash them but
instead created two more can i get a little help?

**Comment by sjha2000** [comment]
can anyone review why the build is failing, please.

**Comment by jayvdb** [comment]
You need to do what I said at https://github.com/coala/coala-quickstart/pull/269#pullrequestreview-139309112

**Comment by MacBox7** [comment]
@sjha2000 waiting for you to complete this.

**Comment by sjha2000** [comment]
the build passed now, what am i missing?
review requested.

**Comment by Naveenaidu** [comment]
@sjha2000 Please fix your commit message.
Your current commit message is 
`Setting.py : Remove unused variables`

The correct Short Log of Head commit should be
`Setting.py: Remove unused variables`

No spaces after :+1:

**Comment by sjha2000** [comment]
squashed all commits.

File: coala_quickstart/generation/Settings.py [hunk 1]
```diff
@@ -52,11 +52,21 @@ def generate_ignore_field(project_dir,
         A comma-separated string containing the globs to ignore.
     """
 
+<<<<<<< HEAD
```
Comment (jayvdb) [inline coala_quickstart/generation/Settings.py]: well this doesnt look right, does it ;-)

edit the file, amend your commit, repush.

**Comment by jayvdb** [comment]
As the bot says at https://github.com/coala/coala-quickstart/pull/269#issuecomment-409262457 , your commit message ends with a `.`.  Please remove it.

**Comment by RaiVaibhav** [comment]
@sjha2000 what problem are you facing ? You can directly ask heLP in channel

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Broken rebase , not fixed for a few days.
Unassigning due to inactivity.

**Pull request closed without merging**